ospd_openvas/daemon.py

```diff
@@ -1154,7 +1159,14 @@ def exec_scan(self, scan_id: str):
                 self.main_db.release_database(kbdb)
                 return
 
-            time.sleep(1)
+            # Wait a second before trying to get result from redis if there
+            # was no results before.
+            # Otherwise, wait 50 msec to give access other process to redis.
+            got_results = False
+            if not got_results:
+                time.sleep(1)
+            time.sleep(0.05)
```

Comment:
Shouldn't be this in an else branch?
```suggestion
            else:
                time.sleep(0.05)
```